src/layers/fp64/scatterplot-layer/scatterplot-layer-64.js

```diff
@@ -18,96 +18,35 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import {Layer} from '../../../lib';
-import {assembleShaders} from '../../../shader-utils';
+import ScatterplotLayer from '../../core/scatterplot-layer';
 import {fp64ify} from '../../../lib/utils/fp64';
-import {GL, Model, Geometry} from 'luma.gl';
 import {readFileSync} from 'fs';
 import {join} from 'path';
 
-const DEFAULT_COLOR = [255, 0, 255, 255];
+export default class ScatterplotLayer64 extends ScatterplotLayer {
 
-const defaultGetPosition = x => x.position;
-const defaultGetRadius = x => x.radius;
-const defaultGetColor = x => x.color || DEFAULT_COLOR;
-
-const defaultProps = {
-  getPosition: defaultGetPosition,
-  getRadius: defaultGetRadius,
-  getColor: defaultGetColor,
-  radius: 30,
-  radiusMinPixels: 0,
-  radiusMaxPixels: Number.MAX_SAFE_INTEGER,
-  drawOutline: false,
-  strokeWidth: 1
-};
-
-export default class ScatterplotLayer64 extends Layer {
-  constructor(props) {
-    super(Object.assign({}, defaultProps, props));
-  }
-
-  initializeState() {
-    const {gl} = this.context;
-    const {attributeManager} = this.state;
-
-    this.setState({
-      model: this.getModel(gl)
-    });
-
-    attributeManager.addInstanced({
-      instancePositionsFP64: {size: 4, update: this.calculateInstancePositions},
-      instanceRadius: {size: 1, update: this.calculateInstanceRadius},
-      layerHeight: {size: 2, update: this.calculateLayerHeight},
-      instanceColors: {
-        size: 4,
-        type: GL.UNSIGNED_BYTE,
-        update: this.calculateInstanceColors
-      }
-    });
-  }
-
-  draw({uniforms}) {
-    this.calculateZoomRadius();
-    this.state.model.render(Object.assign({}, uniforms, {
-      radiusMinPixels: this.props.radiusMinPixels,
-      radiusMaxPixels: this.props.radiusMaxPixels,
-      zoomRadiusFP64: this.state.zoomRadiusFP64
-    }));
-  }
-
-  getShaders() {
+  // Override the super class vertex shader
+  getShaders(id) {
     return {
       vs: readFileSync(join(__dirname, './scatterplot-layer-vertex.glsl'), 'utf8'),
```

Comment:
Do we want to change the name of this file to "scatterplot-layer-vertex-64.glsl"?